test/memory.test.js

```diff
@@ -463,25 +463,88 @@ describe('Memory connector', function() {
       });
     });
 
+    it('automigrate all models - promise variant', function(done) {
+      ds.automigrate()
+        .then(function(result) {
+          done();
+        })
+        .catch(function(err){
+          done(err);
+        });
```

Comment:
FYI: this can be shortened to `.catch(done)`. For the purpose of these tests, I think it's ok to keep the longer version. It's more explicit and thus it may be easier to understand.